test/parallel_api/algorithm/alg.nonmodifying/find_if.pass.cpp

```diff
@@ -54,11 +54,22 @@ struct test_find_if_not
     }
 };
 
+template <typename T, typename Predicate>
+struct NotPred
```

Comment:
We have a similar utility, `TransformOp`, in `copy_if.pass`. Should we make it a general utility?

Reply:
Fixed, thanks

Reply:
Fixed